README.md

```diff
@@ -50,22 +62,22 @@ pipeline:
     observationTimeStamp: '2018-10-11T13:01:02-08:00'
 ```
 
-Step 1 involves copying the source files into a folder:
+First we copy all the source files into a folder:
 ```bash
-mkdir ~/inputs
-cp orthomosaic.tif ~/inputs
-cp plot_shapes.* ~/inputs
-cp experiment.yaml ~/inputs
+mkdir /inputs
+cp orthomosaic.tif /inputs
+cp plot_shapes.* /inputs
+cp experiment.yaml /inputs
 ```
 
-In step 2 we create an folder to hold the output of our processing:
+Next we create an folder to hold the output of our processing:
 ```bash
-mkdir ~/output
+mkdir /output
```

Comment:
Suggestion: Use either exactly the same directory name inside & outside container (e.g. stick to `outputs`, or completely different names. Names which are very similar but not quite the same (`output` vs `outputs`) can lead to confusion and mistakes.

(non-blocking)

Reply:
This has been changed